### DISCUSSION TITLE
https://community.theia-ide.org/ is down

### DISCUSSION THREAD

**Discussion by kittaakos** [eclipse-theia/theia#11265] [General] [answered]
Does anyone know why it is down, or when will it again be available? I noticed that the search functionality was broken on Friday. Today the page does not load. Thank you!

<img width="154" alt="Screen Shot 2022-06-07 at 13 16 43" src="https://user-images.githubusercontent.com/1405703/172366767-8ea3a59e-15d4-4bba-b668-0a38e3b0bffe.png">

**Comment by msujew** [comment, marked as answer]
@kittaakos See https://github.com/eclipse-theia/theia/issues/11251, we're waiting for the owners of the domain to answer.

**Reply by kittaakos** [reply]
Thank you!
